src/transformers/tokenization_utils_base.py

```diff
@@ -605,6 +618,10 @@ def convert_to_tensors(
                 if prepend_batch_axis:
                     value = [value]
 
+                # don't reconvert if it's the right type already
+                if is_tensor(value):
+                    continue
```

Comment:
I'd prefer to use the form `if not is_tensor(value): # then do something`  rather than do nothing, but I let Lysandre decide here.

Reply:
I went ahead and applied your suggestion, @mfuntowicz . Can revert if @LysandreJik prefers the original.